aws/table_aws_rolesanywhere_profile.go

```diff
@@ -0,0 +1,178 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/rolesanywhere"
+
+	rolesanywherev1 "github.com/aws/aws-sdk-go/service/rolesanywhere"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsRolesAnywhereProfile(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_rolesanywhere_profile",
+		Description: "AWS Roles Anywhere Profile",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AnyColumn([]string{"id"}),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFoundException"}),
+			},
+			Hydrate: getProfile,
+			Tags:    map[string]string{"service": "rolesanywhere", "action": "GetProfile"},
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listProfiles,
+			Tags:    map[string]string{"service": "rolesanywhere", "action": "ListProfiles"},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(rolesanywherev1.EndpointsID),
```

Comment:
```suggestion
		GetMatrixItemFunc: SupportedRegionMatrix(AWS_ROLESANYWHERE_SERVICE_ID),
```

We are halfway through removing support for AWS SDK V1 from the plugin, as the V1 SDK has reached its End of Life.
We have listed all the AWS-supported service IDs in the following file: [endpoint_service_ids_gen.go](https://github.com/turbot/steampipe-plugin-aws/blob/main/aws/endpoint_service_ids_gen.go).